pkgs/development/libraries/lzlib/default.nix

```
@@ -0,0 +1,25 @@
+{ lib, stdenv, fetchurl, texinfo }:
+
+stdenv.mkDerivation rec {
+  pname = "lzlib";
+  version = "1.10";
+  outputs = [ "out" "info" ];
+
+  nativeBuildInputs = [ texinfo ];
+
+  src = fetchurl {
+    url = "mirror://savannah/lzip/${pname}/${pname}-${version}.tar.gz";
+    sha256 = "sha256-HWq3gApbQ+Vv0gYH/Sz9qeVQNQ3JX1vrakzhT4W0EEM=";
+  };
+
```

Fix cross-compilation:
```suggestion
  makeFlags = [ "AR:=$(AR)" "CC:=$(CC)" ];

```